ai-machine-learning/thoth-s2i-advise/deploy.md

```diff
@@ -0,0 +1,46 @@
+Now that you are familiar with the configurations our s2i image offers, let's try deploying our sample app using Thoth's S2I build process. 
+
+Assuming you have followed the steps from before are logged in as an admin into `myproject` in the openshift cluster lets deploy - 
```

Comment:
minor fix: Assuming you have followed the steps from before `and` are logged in as an admin...

Reply:
Thanks for pointing that :) Added in - [b2208fe](https://github.com/AICoE/learn-katacoda/pull/21/commits/b2208fe6a73f80d0bc19ff4e63b205fe5c82f1d4)